src/Models/Model.php

```diff
@@ -33,12 +33,15 @@ protected function isTranslationModel(): bool
         return Str::endsWith(get_class($this), 'Translation');
     }
 
-    public function scopePublished($query): Builder
+    public function scopePublished(Builder $query): Builder
```

Comment:
All of the scopes have `Builder $query` in the twill interfaces, I'm actually really surprised php isn't complaining that the signature isn't compatible